reference/nats-protocol/nats-protocol/README.md

```diff
@@ -71,19 +71,34 @@ When using the updated client protocol (see [`CONNECT`](./#connect) below), `INF
+| `tls_required`    | If this is true, then the client must perform the TLS/1.2 handshake. Note, this used to be `ssl_required` and has been updated along with the protocol from SSL to TLS.| bool     | optional |
+| `tls_verify`      | If this is true, the client must provide a valid certificate during the TLS handshake.                                                                                 | bool     | optional |
+| `tls_available`   | If this is true, the client can provide a valid certificate during the TLS handshake.                                                                                  | bool     | optional |
+| `connect_urls`    | List of server urls that a client can connect to.                                                                                                                      | [string] | optional |
+| `ws_connect_urls` | List of server urls that a websocket client can connect to.                                                                                                            | [string] | optional |
+| `ldm`             | If the server supports _Lame Duck Mode_ notifications, and the current server has transitioned to lame duck, `ldm` will be set to `true`.                              | bool     | optional |
+| `git_commit`      | The git hash at which the NATS server was built.                                                                                                                       | string   | optional |
+| `jetstream`       | Whether the server supports JetStream.                                                                                                                                 | bool     | optional |
+| `ip`              | The IP of the server.                                                                                                                                                  | string   | optional |
+| `client_id`       | The ID of the client.                                                                                                                                                  | string   | optional |
+| `client_ip`       | The IP of the client.                                                                                                                                                  | string   | optional |
+| `nonce`           | The nonce for use in CONNECT.                                                                                                                                          | string   | optional |
+| `cluster`         | The name of the cluster.                                                                                                                                               | string   | optional |
+| `cluster_dynamic` | Whether the cluster is dynamic.                                                                                                                                        | bool     | optional |
```

Comment:
`cluster_dynamic` is specific to gateway connections if I recall. This can be removed.